packages/server/modules/activitystream/services/index.js

```diff
@@ -91,14 +91,19 @@ module.exports = {
 
     let dbRawQuery = `
       SELECT act.*
-      FROM stream_acl acl
-      INNER JOIN stream_activity act ON acl."resourceId" = act."streamId"
-      WHERE acl."userId" = ? AND time < ?
+        FROM stream_acl acl
+        INNER JOIN stream_activity act ON acl."resourceId" = act."streamId"
+        WHERE acl."userId" = ? AND time < ?
+      UNION
+      SELECT act.*
+        FROM stream_acl acl
+        INNER JOIN stream_activity act ON acl."userId" = act."userId"
```

Comment:
Do we need a join here? In my tests, where I have multiple streams and multiple rows in `stream_acl` for my user, i get duplicate results